frontend/src/sections/JoinCommunity/JoinCommunity.tsx

```diff
@@ -27,115 +27,135 @@ function Join() {
           </p>
         </div>
 
-        <div className='w-96 md:w-auto mt-10 relative'>
-          <div className='relative '>
-            <Image
-              src='/Phone2.png'
-              alt='Phone Image'
-              width={563}
-              height={707}
-              className='object-contain scale-110'
-            />
-            <div className='absolute left-[72px] rounded-[78px] z-30 w-fit h-fit md:w-[505px] md:h-[707px] inset-1'>
-              <div className='flex flex-col justify-start md:justify-center items-start md:items-center gap-6 md:gap-10 pt-20 md:pt-35 md:mt-8 ml-2 md:ml-0'>
-                <div className='flex flex-row justify-center items-center gap-7 md:gap-10'>
-                  <Link
-                    href='https://www.instagram.com/djangoindia'
-                    target='_blank'
-                    passHref
-                  >
-                    <Image
-                      src='/icons/insta.svg'
-                      alt='Instagram'
-                      width={width > 768 ? 72: 50}
-                      height={width > 768 ? 101 : 50.5}
-                      className='transition transform hover:scale-110'
-                    />
-                  </Link>
+        {/* <div className='md:w-auto mt-10 relative'> */}
+        <div className='relative '>
+          <Image
+            src='/Phone2.png'
+            alt='Phone Image'
+            width={563}
+            height={707}
+            className='object-contain '
+          />
+          <div className='absolute pl-[22%] pr-[9%] md:pl-[10%] top-0 sm:top-[30px] md:top-0 md:left-[72px]  z-30  h-fit  md:h-[707px] inset-1  right-0'>
```

Comment:
I recommend to avoid usage of absolute and relative positioning as much as possible, also using percentage to handle responsiveness won't work. We should be using grid or flex layout with margins and paddings instead.

Reply:
so actually absolute and relative was used to align the icons inside the mobile image so that it won't break for different widths. If you are suggesting to remove the mobile image it can be done then.

Reply:
You can try to align those icons without using absolute and relative positioning. Just think of the the UI without the mobile image in background

Reply:
@Saurabhsharma884, without using  absolute and relative positioning, we can't take icons above the mobile image and as @DevilsAutumn is suggesting mobile image is mandatory it is the only option. I have used grid wherever required. It is looking good now

Reply:
I meant to position the mobile image instead but the current changes will also work and LGTM, Thanks

Reply:
sure, one thing i want to ask can you tell me how do you review the pr's changes in your local?

Reply:
![image (1)](https://github.com/user-attachments/assets/1dc0beb4-608d-4054-8044-12c40880712c)